flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/api/scala/windows.scala

```diff
@@ -85,73 +85,36 @@ object Session {
 }
 
 /**
-  * Helper object for creating a over window.
+  * Helper class for creating an over window. Similar to SQL, over window aggregates compute an
+  * aggregate for each input row over a range of its neighboring rows.
   */
 object Over {
 
   /**
-    * Specifies the time attribute on which rows are grouped.
+    * Specifies the time attribute on which rows are ordered.
     *
-    * For streaming tables call [[orderBy 'rowtime or orderBy 'proctime]] to specify time mode.
+    * For streaming tables, reference a rowtime or proctime time attribute here
+    * to specify the time mode.
     *
     * For batch tables, refer to a timestamp or long attribute.
-    */
-  def orderBy(orderBy: Expression): OverWindowWithOrderBy = {
-    new OverWindowWithOrderBy(Seq[Expression](), orderBy)
-  }
-
-  /**
-    * Partitions the elements on some partition keys.
     *
-    * @param partitionBy some partition keys.
-    * @return A partitionedOver instance that only contains the orderBy method.
+    * @param orderBy field reference
+    * @return an over window with defined order
     */
-  def partitionBy(partitionBy: Expression*): PartitionedOver = {
-    PartitionedOver(partitionBy.toArray)
+  def orderBy(orderBy: Expression): OverWindowPartitionedOrdered = {
+    new OverWindowPartitionedOrdered(Seq[Expression](), orderBy)
```

Comment:
`Seq[Expression]()` -> `Seq()` may be works well. But feel free to keep current state.